server/migrations/versions/2025-11-28-1506_add_dirty_flag_to_customer.py

```diff
@@ -0,0 +1,51 @@
+"""add dirty flag to customer
+
+Revision ID: 856fff9b3dfe
+Revises: f5223877acbb
+Create Date: 2025-11-28 15:06:40.120064
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+# Polar Custom Imports
+
+# revision identifiers, used by Alembic.
+revision = "856fff9b3dfe"
+down_revision = "f5223877acbb"
+branch_labels: tuple[str] | None = None
+depends_on: tuple[str] | None = None
+
+
+def upgrade() -> None:
+    # Add dirty column with default False
+    op.add_column(
+        "customers",
+        sa.Column("dirty", sa.Boolean(), nullable=False, server_default=sa.false()),
```

Comment:
Does the name make sense on the customer? I.e. what does it mean that a customer is dirty outside of processing customer_meters.